.devcontainer/post-create.sh

```diff
@@ -2,20 +2,25 @@
 
 set -e # Fail the whole script on first error
 
-# Fetch Ruby gem dependencies
-bundle install --path vendor/bundle --with='development test'
+# Trust the git repository
+git config --global --add safe.directory /mastodon
 
-# Fetch Javascript dependencies
-yarn install
+# Fetch Ruby gem dependencies
+bundle config path 'vendor/bundle'
+bundle config with 'development test'
+bundle install
 
 # Make Gemfile.lock pristine again
 git checkout -- Gemfile.lock
 
+# Fetch Javascript dependencies
+yarn install
```

Comment:
This could also get changed over to confirm the lockfile like in CI
```suggestion
yarn --frozen-lockfile
```